gdcmSerieHelper.h: No such file or directory

<!--
Thanks for reporting your issue.
Please fill out the sections below.
-->
Issue:
Using this package on ubuntu 18.04 leads to the following error
```
In file included from /home/sirfuser/devel/STIR/src/IO/ITKImageInputFileFormat.cxx:41:0:
/home/sirfuser/miniconda3/include/ITK-5.0/itkGDCMSeriesFileNames.h:25:10: fatal error: gdcmSerieHelper.h: No such file or directory
 #include "gdcmSerieHelper.h"
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
```
My understanding is that this is because I have no system GDCM, and this package does not depend on gdcm, see #1.

```
sudo apt install libgdcm2-dev
```
did not help even ([just like here](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=808401;msg=7)) though this installs `/usr/include/gdcm-2.8/gdcmSerieHelper.h`, which might be an ITK issue?

<br/>
Environment (<code>conda list</code>):
<details>
```
$ conda list
# packages in environment at /home/sirfuser/miniconda3:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
asn1crypto                1.2.0                    py37_0  
beautifulsoup4            4.8.1                    py37_0  
blinker                   1.4                      py37_0  
bzip2                     1.0.8                h7b6447c_0  
ca-certificates           2019.11.28           hecc5488_0    conda-forge
certifi                   2019.11.28               py37_0    conda-forge
cffi                      1.13.2           py37h2e261b9_0  
chardet                   3.0.4                 py37_1003  
conda                     4.8.0                    py37_0    conda-forge
conda-build               3.18.11                  py37_0    conda-forge
conda-forge-pinning       2019.12.02                    0    conda-forge
conda-package-handling    1.6.0            py37h7b6447c_0  
conda-smithy              3.6.3                      py_0    conda-forge
cryptography              2.8              py37h1ba5d50_0  
deprecated                1.2.6                      py_0  
expat                     2.2.6                he6710b0_0  
fftw                      3.3.8           nompi_h7f3a6c3_1110    conda-forge
filelock                  3.0.12                     py_0  
git                       2.23.0          pl526hacde149_0  
gitdb2                    2.0.6                      py_0  
gitpython                 3.0.5                      py_0  
glob2                     0.7                        py_0  
hdf5                      1.10.5          nompi_h3c11f04_1104    conda-forge
idna                      2.8                      py37_0  
isodate                   0.6.0                      py_1  
jinja2                    2.10.3                     py_0  
jpeg                      9c                h14c3975_1001    conda-forge
krb5                      1.16.1               h173b8e3_7  
libarchive                3.3.3                h5d8350f_5  
libcurl                   7.67.0               h20c2e04_0  
libedit                   3.1.20181209         hc058e9b_0  
libffi                    3.2.1                hd88cf55_4  
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_2    conda-forge
libitk                    5.0.1                h2b26915_1    conda-forge
libitk-devel              5.0.1                he1b5a44_1    conda-forge
liblief                   0.9.0                h7725739_2  
libpng                    1.6.37               hed695b0_0    conda-forge
libssh2                   1.8.2                h1ba5d50_0  
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.1.0                h2733197_0  
libxml2                   2.9.9                hea5a465_1  
lz4-c                     1.8.1.2              h14c3975_0  
lzo                       2.10                 h49e0be7_2  
markupsafe                1.1.1            py37h7b6447c_0  
msrest                    0.6.10                     py_0    conda-forge
ncurses                   6.1                  he6710b0_1  
oauthlib                  3.1.0                      py_0  
openssl                   1.1.1d               h516909a_0    conda-forge
patchelf                  0.10                 he6710b0_0  
pcre                      8.43                 he6710b0_0  
perl                      5.26.2               h14c3975_0  
pip                       19.3.1                   py37_0  
pkginfo                   1.5.0.1                  py37_0  
psutil                    5.6.7            py37h7b6447c_0  
py-lief                   0.9.0            py37h7725739_2  
pycosat                   0.6.3            py37h14c3975_0  
pycparser                 2.19                     py37_0  
pycrypto                  2.6.1            py37h14c3975_9  
pygithub                  1.43.6                   py37_0    conda-forge
pyjwt                     1.7.1                    py37_0  
pyopenssl                 19.1.0                   py37_0  
pysocks                   1.7.1                    py37_0  
python                    3.7.4                h265db76_1  
python-libarchive-c       2.8                     py37_13  
pytz                      2019.3                     py_0  
pyyaml                    5.1.2            py37h7b6447c_0  
readline                  7.0                  h7b6447c_5  
requests                  2.22.0                   py37_1  
requests-oauthlib         1.3.0                      py_0  
ripgrep                   0.10.0               hc07d326_0  
ruamel.yaml               0.16.5           py37h7b6447c_1  
ruamel.yaml.clib          0.2.0            py37h7b6447c_0  
ruamel_yaml               0.15.46          py37h14c3975_0  
setuptools                42.0.2                   py37_0  
six                       1.13.0                   py37_0  
smmap2                    2.0.5                    py37_0  
soupsieve                 1.9.5                    py37_0  
sqlite                    3.30.1               h7b6447c_0  
tk                        8.6.8                hbc83047_0  
toolz                     0.10.0                     py_0  
tqdm                      4.40.0                     py_0  
urllib3                   1.25.7                   py37_0  
vsts-python-api           0.1.22                     py_0    conda-forge
wheel                     0.33.6                   py37_0  
wrapt                     1.11.2           py37h7b6447c_0  
xz                        5.2.4                h14c3975_4  
yaml                      0.1.7                had09818_2  
zlib                      1.2.11               h7b6447c_3  
zstd                      1.3.7                h0b5b093_0  
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : base
    active env location : /home/sirfuser/miniconda3
            shell level : 1
       user config file : /home/sirfuser/.condarc
 populated config files : /home/sirfuser/.condarc
          conda version : 4.8.0
    conda-build version : 3.18.11
         python version : 3.7.4.final.0
       virtual packages : __glibc=2.27
       base environment : /home/sirfuser/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/sirfuser/miniconda3/pkgs
                          /home/sirfuser/.conda/pkgs
       envs directories : /home/sirfuser/miniconda3/envs
                          /home/sirfuser/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.0 requests/2.22.0 CPython/3.7.4 Linux/4.15.0-29-generic ubuntu/18.04.2 glibc/2.27
                UID:GID : 1001:1001
             netrc file : None
           offline mode : False
```
</details>

--- a/.azure-pipelines/azure-pipelines-osx.yml
+++ b/.azure-pipelines/azure-pipelines-osx.yml
@@ -5,7 +5,7 @@
 jobs:
 - job: osx
   pool:
-    vmImage: macOS-10.13
+    vmImage: macOS-10.14
   timeoutInMinutes: 360
   strategy:
     maxParallel: 8
@@ -42,7 +42,7 @@ jobs:
       source activate base
       echo "Configuring conda."
 
-      setup_conda_rc ./ ./recipe ./.ci_support/${CONFIG}.yaml
+      setup_conda_rc ./ "./recipe" ./.ci_support/${CONFIG}.yaml
       export CI=azure
       source run_conda_forge_build_setup
       conda update --yes --quiet --override-channels -c conda-forge -c defaults --all
@@ -53,24 +53,24 @@ jobs:
 
   - script: |
       source activate base
-      mangle_compiler ./ ./recipe ./.ci_support/${CONFIG}.yaml
+      mangle_compiler ./ "./recipe" ./.ci_support/${CONFIG}.yaml
     displayName: Mangle compiler
 
   - script: |
       source activate base
-      make_build_number ./ ./recipe ./.ci_support/${CONFIG}.yaml
+      make_build_number ./ "./recipe" ./.ci_support/${CONFIG}.yaml
     displayName: Generate build number clobber file
 
   - script: |
       source activate base
-      conda build ./recipe -m ./.ci_support/${CONFIG}.yaml --clobber-file ./.ci_support/clobber_${CONFIG}.yaml
+      conda build "./recipe" -m ./.ci_support/${CONFIG}.yaml --clobber-file ./.ci_support/clobber_${CONFIG}.yaml
     displayName: Build recipe
 
   - script: |
       source activate base
       export GIT_BRANCH=$BUILD_SOURCEBRANCHNAME
-      upload_package ./ ./recipe ./.ci_support/${CONFIG}.yaml
+      upload_package ./ "./recipe" ./.ci_support/${CONFIG}.yaml
     displayName: Upload package
     env:
       BINSTAR_TOKEN: $(BINSTAR_TOKEN)
-    condition: not(eq(variables['UPLOAD_PACKAGES'], 'False'))
+    condition: and(succeeded(), not(eq(variables['UPLOAD_PACKAGES'], 'False')))

--- a/.azure-pipelines/azure-pipelines-win.yml
+++ b/.azure-pipelines/azure-pipelines-win.yml
@@ -10,8 +10,8 @@ jobs:
   strategy:
     maxParallel: 4
     matrix:
-      win_c_compilervs2015cxx_compilervs2015vc14:
-        CONFIG: win_c_compilervs2015cxx_compilervs2015vc14
+      win_:
+        CONFIG: win_
         CONDA_BLD_PATH: D:\\bld\\
         UPLOAD_PACKAGES: True
   steps:
@@ -58,17 +58,18 @@ jobs:
       inputs:
         packageSpecs: 'python=3.6 conda-build conda conda-forge::conda-forge-ci-setup=2' # Optional
         installOptions: "-c conda-forge"
-        updateConda: false
+        updateConda: true
       displayName: Install conda-build and activate environment
 
     - script: set PYTHONUNBUFFERED=1
 
     # Configure the VM
-    - script: setup_conda_rc .\ .\recipe .\.ci_support\%CONFIG%.yaml
+    - script: setup_conda_rc .\ ".\recipe" .\.ci_support\%CONFIG%.yaml
 
     # Configure the VM.
     - script: |
         set "CI=azure"
+        call activate base
         run_conda_forge_build_setup
       displayName: conda-forge build setup
     
@@ -80,7 +81,8 @@ jobs:
 
     # Special cased version setting some more things!
     - script: |
-        conda.exe build recipe -m .ci_support\%CONFIG%.yaml
+        call activate base
+        conda.exe build "recipe" -m .ci_support\%CONFIG%.yaml
       displayName: Build recipe (vs2008)
       env:
         VS90COMNTOOLS: "C:\\Program Files (x86)\\Common Files\\Microsoft\\Visual C++ for Python\\9.0\\VC\\bin"
@@ -88,7 +90,8 @@ jobs:
       condition: contains(variables['CONFIG'], 'vs2008')
 
     - script: |
-        conda.exe build recipe -m .ci_support\%CONFIG%.yaml
+        call activate base
+        conda.exe build "recipe" -m .ci_support\%CONFIG%.yaml
       displayName: Build recipe
       env:
         PYTHONUNBUFFERED: 1
@@ -96,8 +99,9 @@ jobs:
 
     - script: |
         set "GIT_BRANCH=%BUILD_SOURCEBRANCHNAME%"
-        upload_package .\ .\recipe .ci_support\%CONFIG%.yaml
+        call activate base
+        upload_package .\ ".\recipe" .ci_support\%CONFIG%.yaml
       displayName: Upload package
       env:
         BINSTAR_TOKEN: $(BINSTAR_TOKEN)
-      condition: not(eq(variables['UPLOAD_PACKAGES'], 'False'))
+      condition: and(succeeded(), not(eq(variables['UPLOAD_PACKAGES'], 'False')))

--- a/.ci_support/linux_.yaml
+++ b/.ci_support/linux_.yaml
@@ -23,7 +23,7 @@ jpeg:
 libpng:
 - '1.6'
 libtiff:
-- 4.0.10
+- 4.1.0
 pin_run_as_build:
   expat:
     max_pin: x.x

--- a/.ci_support/osx_.yaml
+++ b/.ci_support/osx_.yaml
@@ -23,7 +23,7 @@ jpeg:
 libpng:
 - '1.6'
 libtiff:
-- 4.0.10
+- 4.1.0
 macos_machine:
 - x86_64-apple-darwin13.4.0
 macos_min_version:

--- a/.ci_support/win_.yaml
+++ b/.ci_support/win_.yaml
@@ -1,11 +1,11 @@
 c_compiler:
-- vs2015
+- vs2017
 channel_sources:
 - conda-forge,defaults
 channel_targets:
 - conda-forge main
 cxx_compiler:
-- vs2015
+- vs2017
 fftw:
 - '3'
 jpeg:
@@ -13,7 +13,7 @@ jpeg:
 libpng:
 - '1.6'
 libtiff:
-- 4.0.10
+- 4.1.0
 pin_run_as_build:
   fftw:
     max_pin: x
@@ -23,11 +23,5 @@ pin_run_as_build:
     max_pin: x.x
   libtiff:
     max_pin: x
-  vc:
-    max_pin: x
 vc:
 - '14'
-zip_keys:
-- - vc
-  - c_compiler
-  - cxx_compiler

--- a/README.md
+++ b/README.md
@@ -43,10 +43,10 @@ Current build status
                 </a>
               </td>
             </tr><tr>
-              <td>win_c_compilervs2015cxx_compilervs2015vc14</td>
+              <td>win</td>
               <td>
                 <a href="https://dev.azure.com/conda-forge/feedstock-builds/_build/latest?definitionId=5006&branchName=master">
-                  <img src="https://dev.azure.com/conda-forge/feedstock-builds/_apis/build/status/libitk-feedstock?branchName=master&jobName=win&configuration=win_c_compilervs2015cxx_compilervs2015vc14" alt="variant">
+                  <img src="https://dev.azure.com/conda-forge/feedstock-builds/_apis/build/status/libitk-feedstock?branchName=master&jobName=win&configuration=win_" alt="variant">
                 </a>
               </td>
             </tr>
@@ -107,7 +107,7 @@ A feedstock is made up of a conda recipe (the instructions on what and how to bu
 the package) and the necessary configurations for automatic building using freely
 available continuous integration services. Thanks to the awesome service provided by
 [CircleCI](https://circleci.com/), [AppVeyor](https://www.appveyor.com/)
-and [TravisCI](https://travis-ci.org/) it is possible to build and upload installable
+and [TravisCI](https://travis-ci.com/) it is possible to build and upload installable
 packages to the [conda-forge](https://anaconda.org/conda-forge)
 [Anaconda-Cloud](https://anaconda.org/) channel for Linux, Windows and OSX respectively.
 

--- a/recipe/libitk-devel_install.bat
+++ b/recipe/libitk-devel_install.bat
@@ -2,5 +2,5 @@ set BUILD_DIR=%SRC_DIR%\bld
 cmake -DCOMPONENT=Development -P %BUILD_DIR%\cmake_install.cmake
 cmake -DCOMPONENT=DebugDevel -P %BUILD_DIR%\cmake_install.cmake
 cmake -DCOMPONENT=cpplibraries  -P %BUILD_DIR%\cmake_install.cmake
-
+cmake -DCOMPONENT=Headers -P %BUILD_DIR%\cmake_install.cmake
 

--- a/recipe/libitk-devel_install.sh
+++ b/recipe/libitk-devel_install.sh
@@ -1,3 +1,3 @@
 BUILD_DIR=${SRC_DIR}/build
 cmake -DCOMPONENT=Development -P ${BUILD_DIR}/cmake_install.cmake
-
+cmake -DCOMPONENT=Headers -P ${BUILD_DIR}/cmake_install.cmake

